### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [dmorin89/prj-rev-bwfs-dasmoto#5]
When adding an image to a site we usually always try to include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) for the image because it boosts search engine results, increases the [accessibility](https://www.codecademy.com/articles/accessibility) of our site, and it is used as a backup, in case the image cannot be loaded. 

For example, you could include something like the following:
`<img src="https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-2/hacksaw.jpeg" alt ="Hacksaw">`

https://github.com/dmorin89/prj-rev-bwfs-dasmoto/blob/1658be7cc0688c33db5a19a3159b9746b6e870e2/Dasmoto's%20Arts%26Crafts/index.html#L12
